test/verify/check-services

```diff
@@ -188,6 +189,281 @@ WantedBy=default.target
         b.go('#/foo')
         b.wait_text('#exists', 'false')
 
+    def testCreateTimer(self):
+        m = self.machine
+        b = self.browser
+
+        def svc_sel(service):
+            return 'tr[data-goto-unit="%s"]' % service
+        def wait_systemctl_timer(time):
+            with testvm.Timeout(seconds=20, error_message="Timeout while waiting for systemctl list-timers"):
```

Comment:
Timeout is patched now, you can add `machine=m` to make sure it actually works with the ssh command